subjects/ascii-art.en.md

```diff
@@ -0,0 +1,64 @@
+## ascii-art
+
+### Objectives
+
+Ascii-art consists on receiving a `string` has an argument and outputing the `string` in a graphic representation of ascii letters and numbers.
```

Comment:
ASCII letters & numbers means every other ASCII characters should be ignored ?

Reply:
should we include the other characters ?

Reply:
I don't know, the space maybe 😄

Reply:
should we could include other characters?